source/scripts/Provider/EthProvider.ts

```diff
@@ -27,6 +27,19 @@ export const EthProvider = (host: string) => {
 
     if (!decodedTx) throw cleanErrorStack(ethErrors.rpc.internal());
 
+    console.log('params', params);
+
+    if ((tx.type && tx.type === '0x0') || (!tx.type && !tx.data)) {
```

Comment:
Remove the tx type check here as we use this method for any sendTransaction case